.github/workflows/ci-js.yml

```diff
@@ -77,8 +74,7 @@ jobs:
       - name: Run create-turbo tests
         # `pnpm --` calls the `turbo` script in the root package.json
         # --filter and --color args are for turbo CLI
-        # -- --runInBand is passed forward to jest (which is in packages/create-turbo package.json `test` script)
-        run: pnpm -- turbo run test --filter=create-turbo --color -- --runInBand
+        run: pnpm -- turbo run test --filter=create-turbo --color
```

Comment:
Glad to see this go (although not sure if it will cause more timeout issues). I did see an improvement in CI with this flag as noted here: https://stackoverflow.com/a/44506948/986415

Reply:
This is a "play with concurrency in Jest" setup thing. I _believe_ that the concurrency thing is a net win; but I'm not sure.